### PR TITLE
Implemented Observable#toCompletable

### PR DESCRIPTION
**Pull request by stealthcode**

### CODE REVIEW

**Comment by stealthcode** [comment]
Addresses part of #3443

File: src/main/java/rx/Observable.java [hunk 1]
```diff
@@ -232,6 +232,26 @@ public void call(Subscriber<? super R> o) {
         return new Single<T>(OnSubscribeSingle.create(this));
     }
 
+    /**
+     * Returns a Completable that emits the single item emitted by the source Observable, if that Observable
```
Comment (akarnokd) [inline src/main/java/rx/Observable.java]: There are no items involved with Completable.

File: src/main/java/rx/Observable.java [hunk 2]
```diff
@@ -232,6 +232,26 @@ public void call(Subscriber<? super R> o) {
...
+     *  <dd>{@code toCompletable} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     *
+     * @return a Completable that calls onCompleted on it's subscriber when the source Observable calls onCompleted
+     * @see <a href="http://reactivex.io/documentation/completable.html">ReactiveX documentation: Completable</a>
+     * @since (if this graduates from Experimental/Beta to supported, replace this parenthetical with the release number)
+     */
+    @Experimental
+    public Completable toCompletable() {
+        return new Completable(OnSubscribeCompletable.create(this));
```
Comment (akarnokd) [inline src/main/java/rx/Observable.java]: You can use `Completable.fromObservable()`.

**Comment by akarnokd** [comment]
:+1:

**Comment by akarnokd** [comment]
```
rx.internal.operators.OnSubscribeCompletableTest > testShouldUseUnsafeSubscribeInternallyNotSubscribe FAILED
    java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:92)
        at org.junit.Assert.assertTrue(Assert.java:43)
        at org.junit.Assert.assertFalse(Assert.java:68)
        at org.junit.Assert.assertFalse(Assert.java:79)
        at rx.internal.operators.OnSubscribeCompletableTest.testShouldUseUnsafeSubscribeInternallyNotSubscribe(OnSubscribeCompletableTest.java:96)
```

The `Completable.fromObservable` needs to change to `unsafeSubscribe`.

**Comment by stealthcode** [comment]
Thanks @akarnokd

**Comment by akarnokd** [comment]
:+1:

**Comment by stevegury** [comment]
:+1:

**Pull request merged**